### PR TITLE
Hotfix/change system calls to filesystem functions

### PR DESCRIPTION
**Pull request by Stretch96**
Issue: https://github.com/dxw/whippet/issues/12

Adds a couple of functions in whippet_helpers to replicate `rm -r` and `cp -a`, and replaces the use of `system()` with PHP filesystem functions.

### CODE REVIEW

**Comment by tomdxw** [comment]
Testing now. This seems to work just fine.

However `whippet deploy` seems to run about 90 seconds slower. I'll create a separate issue about this as this PR works: #40.

File: lib/modules/helpers/whippet_helpers.trait.php [hunk 1]
```diff
@@ -80,6 +80,45 @@ function find_file($file, $include_dir = false){
     return false;
   }
 
+  // 77.079482% credit:
+  // gimmicklessgpt@gmail.com
+  // http://php.net/manual/en/function.copy.php
+  // Modified to copy symlinks
+  function recurse_copy($src,$dst) {
+    $dir = opendir($src);
+    @mkdir($dst);
```
Comment (tomdxw) [inline lib/modules/helpers/whippet_helpers.trait.php]: `@` is used to hide warnings. We don't want any warnings, hidden or visible.

**Comment by tomdxw** [comment]
@Stretch96 Looks good apart from one use of `@`. Happy to merge once that's been removed.

**Comment by Stretch96** [comment]
@tomdxw Changed the `@mkdir` with a conditional to check if it's already a directory or link

**Comment by tomdxw** [comment]
👍

**Pull request merged**